### ISSUE TITLE
Multiple DHCP records  after change guest VM IP address

### ISSUE THREAD

**Issue by havengit** [apache/cloudstack#5058]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
VR
~~~

##### CLOUDSTACK VERSION

~~~
4.15.0.0
~~~

##### CONFIGURATION

- Advanced Networking 
- L3 Network
- Isolated
- Shared

--

##### STEPS TO REPRODUCE

~~~

1. shutdown Guest VM
2. Change IP address for NIC
3. start Guest VM
4. VM will get previous IP again 
5.  In VR , dnsmasq config has two dhcp record.

> cat /etc/dhcphosts.txt  |grep TC-APP09CE
1e:00:aa:00:00:6a,10.226.18.169,TC-APP09CE,infinite
1e:00:aa:00:00:6a,10.226.18.189,TC-APP09CE,infinite


~~~

##### EXPECTED RESULTS

~~~
after change IP addreses for guest VM. remove  previous IP from dnsmasq record. The  VM will get New IP
~~~

##### ACTUAL RESULTS

~~~
VM get previous IP from dhcp server
~~~

**Comment by rhtyd** [comment]
@havengit how did you change the IP address of nic? Or did you add adding secondary IPs to the nic?

**Comment by rhtyd** [comment]
cc @Pearl1594 @weizhouapache

**Comment by weizhouapache** [comment]
@rhtyd @havengit 
I can reproduce the issue. I will work on it

**Comment by rhtyd** [comment]
Fantastic thanks @weizhouapache

**Comment by havengit** [comment]
> @havengit how did you change the IP address of nic? Or did you add adding secondary IPs to the nic?

no secondary IPs 
From UI ,instance  --> nic --> change ip address

**Comment by rhtyd** [comment]
Fixed by #5059
